src/west/commands/west-commands-schema.yml

```diff
@@ -4,51 +4,12 @@
 ##
 ## By convention, this file is named west-commands.yml.
 
-# The top-level of the file is a map with a 'west-commands:' key.  The
-# west-commands key's value is a sequence of "command descriptors".
-# Each command descriptor gives the location of a file containing west
-# commands, along with the names of the commands provided by the file
-# (and optionally the names of the classes which define them).
+# --------------------------------------------------------------
 #
-# Here is an example west-commands.yml file:
+#  DO NOT CHANGE THIS FILE WITHOUT UPDATING THE DOCUMENTATION!
```

Comment:
If someone comes in after us, and needs to maintain `west`, then I think a reference to the documentation would be better.
So instead of 
> `#  DO NOT CHANGE THIS FILE WITHOUT UPDATING THE DOCUMENTATION!`

to something like

> `#  DO NOT CHANGE THIS FILE WITHOUT UPDATING THE WEST DOCUMENTATION FOUND IN ZEPHYR !`
> `# <zephyr-repo>/doc/guides/west/repo-tool.rst`

As it might not be obviously exactly what docs to update.
Or do you actually mean some other docs ?

Although, I haven't found the place where we actually document the yml schema.
See also other comment.

Reply:
I did this once before, but then upstream Zephyr moved all the documentation around on me :(. So now I am hesitant to do this again, but I will give it another try.

There is a chicken and egg problem since the documentation is being added to Zephyr in files that do not exist yet in the branch that I linked in my other comment. Can we perhaps add the links after we have merged the documentation? That will be too late for west v0.5.7, but that's OK, because master is where changes to this file would happen.

Reply:
> I did this once before, but then upstream Zephyr moved all the documentation around on me :(. So now I am hesitant to do this again, but I will give it another try.

I understand, but then may just refer to Zephyr docs in broad terms, so someone updating  this knows that the docs to update is not in this repo.
But let's add that, when making a new commit, as this is low prio.